src/bin/cql-stress-cassandra-stress/java_generate/values/varint.rs

```diff
@@ -0,0 +1,29 @@
+use scylla::frame::{response::result::CqlValue, value::CqlVarint};
+
+use crate::java_generate::distribution::Distribution;
+
+use super::{ValueGenerator, ValueGeneratorFactory};
+
+#[derive(Default)]
+pub struct VarInt;
+
+impl ValueGenerator for VarInt {
+    fn generate(
+        &mut self,
+        identity_distribution: &mut dyn Distribution,
+        _size_distribution: &mut dyn Distribution,
+    ) -> CqlValue {
+        // BigInt::from_i64 returns Some(). This unwrap is safe.
+        CqlValue::Varint(CqlVarint::from_signed_bytes_be(
+            identity_distribution.next_i64().to_be_bytes().to_vec(),
+        ))
+    }
```

Comment:
Comment refers to `unwrap`, but I can't see any.

Reply:
Nit: `from_signed_bytes_be_slice` is the constructor intended for such a use case - no need for `to_vec` then.

Reply:
> Comment refers to `unwrap`, but I can't see any.

It's a remnant that I forgot to remove. First version would use a `num_bigint` crate and its `BigInt` struct. However, the dependency is not needed since we can easily represent `i64` in a native `varint` representation (a signed integer's representation in big-endian order bytes).